web/config/initializers/shopify_app.rb

```diff
@@ -37,7 +37,7 @@
 
   config.api_key = ENV.fetch("SHOPIFY_API_KEY", "").presence
   config.secret = ENV.fetch("SHOPIFY_API_SECRET", "").presence
-  config.myshopify_domain = ENV.fetch("SHOP_CUSTOM_DOMAIN", "").presence
+  config.myshopify_domain = ENV.fetch("SHOP_CUSTOM_DOMAIN", "").presence if ENV.fetch("SHOP_CUSTOM_DOMAIN", "").present?
```

Comment:
Is it at all possible to use `HOST` as the name of this environment variable? Would be great if we could use that naming / implementation here instead of introducing a new variable that sounds like it does the same thing 🤔

Reply:
I think that the host url of the service is not the same url than the shop url

Reply:
coolio, then to the moon with this PR!